pkg/cache/cluster.go

```diff
@@ -1071,6 +1073,19 @@ func (c *clusterCache) GetClusterInfo() ClusterInfo {
 	}
 }
 
+// GetClusterInfoSnapshot returns cluster cache statistics
```

Comment:
I would like to see some more details in the comments that these variables are normally held by a lock and that under they can not be modified in this function at all etc. Just to make it more clear that these are normally lock protected.

Reply:
fixed.